.github/workflows/ruby.yml

```diff
@@ -69,6 +70,16 @@ jobs:
             gemfile: rails_6_1
           - ruby: 2.4
             gemfile: rails_6_1
+          - ruby: 2.2
+            gemfile: rails_7_0
+          - ruby: 2.3
+            gemfile: rails_7_0
+          - ruby: 2.4
+            gemfile: rails_7_0
+          - ruby: 2.5
+            gemfile: rails_7_0
+          - ruby: 2.6
+            gemfile: rails_7_0
```

Comment:
We shouldn't be testing on Ruby versions that are not supported by Rails 7. Rails 7 only supports 2.7+

Reply:
👋 I believe this is the exclusion list in the `.yml` so it won't run on those versions. 

Given on the discussion [here](https://github.com/rails/jbuilder/pull/526#issuecomment-1028934958) and that this PR has not been updated recently I've included Rails 7 and Ruby 3.1 in a single PR [here](https://github.com/rails/jbuilder/pull/528) that originally had only Ruby 3.1 testing support.